Mathlib/Order/Atoms.lean

```diff
@@ -352,14 +356,108 @@ theorem isCoatomic_iff_forall_isCoatomic_Ici [OrderTop α] :
       forall_congr' fun _ => isCoatomic_dual_iff_isAtomic.symm.trans Iff.rfl
 #align is_coatomic_iff_forall_is_coatomic_Ici isCoatomic_iff_forall_isCoatomic_Ici
 
+section StronglyAtomic
+
+variable {α : Type*} {a b : α} [Preorder α]
+
+/-- An order is strongly atomic if every nontrivial interval `[a,b]`
+contains an element covering `a`. -/
+@[mk_iff]
+class IsStronglyAtomic (α : Type*) [Preorder α] : Prop where
+  (exists_covBy_le_of_lt : ∀ (a b : α), a < b → ∃ x, a ⋖ x ∧ x ≤ b)
+
+theorem LT.lt.exists_covby_le [IsStronglyAtomic α] (h : a < b) : ∃ x, a ⋖ x ∧ x ≤ b :=
```

Comment:
Can you make sure this theorem is also accessible via a non-dot notation name, eg `exists_covBy_le_of_lt`?